src/appmain.ts

```diff
@@ -246,14 +253,11 @@ export class ReplayWebApp extends LitElement {
   }
 
   renderNavBrand() {
-    return html` <img
-        style="height: 1.25rem; user-drag: none; user-select: none; -moz-user-select: none;"
-        src="data:image/svg+xml,${encodeURIComponent(
-          brandLockupColor as string,
-        )}"
-        alt="ReplayWeb.page"
-        draggable="false"
-      />
+    return html` <div class="text-left">
+        <svg width="9.5rem" height="1.25rem">
+          ${unsafeSVG(brandLockupColor)}
```

Comment:
Nice!  Specifying it must be a string stops the linter from being upset.  Probably worth it?  No strong opinions personally, just something I ran into and did before.  ¯\\\_(ツ)_/¯ 
```suggestion
          ${unsafeSVG(brandLockupColor as string)}
```

Reply:
Just replaced it with `fa-icon` as it has support for custom width/height